Makefile.core.mk

```diff
@@ -361,12 +361,38 @@ gen: mod-download-go go-gen mirror-licenses format update-crds operator-proto co
 gen-check: gen check-clean-repo
 
 copy-templates:
+	# gateway charts
 	rm -r manifests/charts/gateways/istio-egress/templates
 	mkdir manifests/charts/gateways/istio-egress/templates
 	cp -r manifests/charts/gateways/istio-ingress/templates/* manifests/charts/gateways/istio-egress/templates
 	find ./manifests/charts/gateways/istio-egress/templates -type f -exec sed -i -e 's/ingress/egress/g' {} \;
 	find ./manifests/charts/gateways/istio-egress/templates -type f -exec sed -i -e 's/Ingress/Egress/g' {} \;
 
+	# external istiod remote cluster charts
+	cp manifests/charts/base/templates/reader-serviceaccount.yaml manifests/charts/istiod-remote/templates
+	cp manifests/charts/istio-control/istio-discovery/templates/mutatingwebhook.yaml manifests/charts/istiod-remote/templates
+	cp manifests/charts/istio-control/istio-discovery/templates/reader-clusterrole.yaml manifests/charts/istiod-remote/templates
+	cp manifests/charts/istio-control/istio-discovery/templates/reader-clusterrolebinding.yaml manifests/charts/istiod-remote/templates
+
+	# external istiod config cluster charts
+	cp manifests/charts/istio-control/istio-discovery/files/injection-template.yaml manifests/charts/istiod-remote/files
+	cp manifests/charts/istio-control/istio-discovery/files/gateway-injection-template.yaml manifests/charts/istiod-remote/files
+	cp manifests/charts/istio-control/istio-discovery/templates/istiod-injector-configmap.yaml manifests/charts/istiod-remote/templates
+	cp manifests/charts/istio-control/istio-discovery/templates/configmap.yaml manifests/charts/istiod-remote/templates
+	sed -e '1 i {{- if .Values.global.configCluster }}' -e '$$ a {{- end }}' manifests/charts/base/crds/crd-all.gen.yaml > manifests/charts/istiod-remote/templates/crd-all.gen.yaml
+	# sed -e '1 i {{- if .Values.global.configCluster }}' -e '$$ a {{- end }}' manifests/charts/base/crds/crd-operator.yaml > manifests/charts/istiod-remote//templates/crd-operator.yaml
```

Comment:
why is this commented out? can we add a comment on why this `sed` is needed?

Reply:
I wasn't sure it's needed. It's not needed in the sample instructions since we use `manifest generate` and it also seems that `istioctl install` will add it if it's not already there, so the only case it would be needed here is if we want to run the operator on the remote cluster to to install the remote profile. I wasn't sure if that makes sense, so I left a commented out line here, but thinking about it, I'd be inclined to just remove it for now. if it's needed in the future, we can add it back. WDYT?

Reply:
On second thought, what's the harm of installing one more CRD that may or may not be needed. I'll uncomment the line.